tests/integration/suite/test_system_project.py

```diff
@@ -71,3 +73,14 @@ def test_system_project_cant_be_deleted(admin_mc, admin_cc):
         admin_mc.client.delete(system_project)
     assert e.value.error.status == 405
     assert e.value.error.message == 'System Project cannot be deleted'
+
+
+def test_system_namespaces_default_svc_account(admin_mc):
+    k8sclient = CoreV1Api(admin_mc.k8s_client)
+    saccounts = k8sclient.list_service_account_for_all_namespaces()
```

Comment:
This func can accept a `field_selector` to filter the results to just the name required which would save the loop and name check below